platform/gtk4/cog-platform-gtk4.c

```diff
@@ -84,15 +84,17 @@ static struct platform_window win = {
     .device_scale_factor = 1,
 };
 
-static const char s_vertex_shader[] = "attribute vec2 pos;\n"
+static const char s_vertex_shader[] = "#version 100\n"
```

Comment:
As discussed in a chat conversation, all ES contexts are expected to support GLSL 1.0, so this is expected to always work regardless of the driver/hardware combination. These shaders are simple enough that they do not use any feature outside of version 1.0—all good!

OTOH, for the sake of correctness we should add the same to the similar (if not same) shaders in `cog-platform-x11.c` and `cog-drm-gles-renderer.c`. It would be fine to do so in a follow-up patch, so we unblock people wanting to use the GTK4 plugin ASAP.

Reply:
I have made all the platform plug-ins use exactly the same shaders and textured quad painting code in #392 😸